### DISCUSSION TITLE
Set return url on logout

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3967] [Help] [answered]
Hi,

I have a public page with a button that navigate to a second page that required permission and this works perfectly fine and I can see that after login it gets redirected to the return url.
On this second page, I have a button to logout and I see that it redirects back to `/builder/app/login`. I would like to have it redirect back to the app public page.
I tried setting a `Navigate to` action this in the button action, but it doesn't seems to work. Is there a way around this?

<img width="576" alt="Screenshot 2022-01-07 at 14 47 46" src="https://user-images.githubusercontent.com/7136209/148556054-e9ca4662-181f-4d99-bda9-9eb95a206933.png">

Thanks!

**Comment by mjashanks** [comment, marked as answer]
Hmm, I see what you mean - `Log Out` doesn't seem to work. I'll have to log an issue for this :/

**Reply by micebrain** [reply]
Okie, I will wait for a fix :)
